Sources/ShopifyCheckoutSheetKit/CheckoutWebView.swift

```diff
@@ -38,12 +38,15 @@ class CheckoutWebView: WKWebView {
 
 	private static var cache: CacheEntry?
 	static var preloadingActivatedByClient: Bool = false
+	// a ref to the view is needed when preload is deactivated in order to detatch bridge
+	static weak var uncacheableViewRef: CheckoutWebView?
+	var isBridgeAttached = false
```

Comment:
THis flag is only intended to be used in test. open to alternatives, but as mentioned in the description, testing this is quite challenging